### DISCUSSION TITLE
Unable to connect NRF24L01+

### DISCUSSION THREAD

**Discussion by unknown0816** [tbnobody/OpenDTU#1008] [Q&A] [answered]
Hi,

I have following situation: I flashed OpenDTU succesful to my ESP32 Board and could include the device into my WLAN and so on. No Problem. But as soon as I connect/wire the NRF Module the whole ESP32 just freezes. Meaning no red blinking, no web access to the UI and so on. If I disconnect the NRF (by pulling 3V or GND) the ESP32 works fine again. 
This Freezing happens if I connect the NRF before powering the ESP32 and during normal processing. I already tested like 10 cables and currently the whole thing is connected to a 5V/3A Adapter. So I guess this shouldn't be a problem. I double and triple checked the wires, but I cannot find any problem.

Does somebody know this behaviour? Or is the NRF module broken?

**Comment by DPO99** [comment]
Well should be a Problem with the NRf :). Please Double chekx wiring. If still exists buy a ebyte nrf (good quality).

**Comment by Omega13x** [comment]
Did you notice that unlike the instructions, the pins are on the back of the NRF? So mirrored.

**Reply by DPO99** [reply]
The wireing ist Just an example. You should Always Check the Pin Info in the NRf.

**Reply by unknown0816** [reply]
Thanks for the answers. Unfortunately, on the NRF are no Pin Information. And as far as I understand the instructions I'm looking from a side where leds, the antenna and so on are visible. To connect the Pins I have to turn around the NRF and have mirrored pins.

**Reply by Omega13x** [reply, marked as answer]
It is exactly like that.
![image](https://github.com/tbnobody/OpenDTU/assets/123419851/b6414345-86c5-47af-bfb4-8eab06a363ed)
![image](https://github.com/tbnobody/OpenDTU/assets/123419851/2b3e661d-0414-44c7-9a92-29f5065eb5c6)
If it still doesn't work, the NRF is probably broken.

**Reply by unknown0816** [reply]
Yep. It was.. I brought a new one. And it worked instantly ;) Thank you for your help.
